travis.py

```diff
@@ -116,7 +116,7 @@ def run_command(command):
         run_args = "%s --privileged" % run_args
     
     if language == "ansible":
-        build_args = "%s --build-arg ANSIBLE_VERSION=%s" % (build_args, os.environ.get("ANSIBLE_VERSION"))
+        build_args = "%s --build-arg ANSIBLE_VERSION=%s --build-arg PYTHON_VERSION=%s --build-arg GLIBC_VERSION=%s" % (build_args, os.environ.get("ANSIBLE_VERSION"), os.environ.get("PYTHON_VERSION"), os.environ.get("GLIBC_VERSION"))
```

Comment:
`ANSIBLE_VERSION` is not needed anymore, since Ansible is configured using `VERSION`.

Reply:
Yeah missed that one the .py file